drivers/sensor/shtcx/shtcx.c

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright (c) 2021 Thomas Stranger
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT sensirion_shtcx
```

Comment:
Did you lose shtcx bindings during rebasing?

Reply:
Indeed, thanks. This is also the cause of the failing ci tests.